src/controllers/auth.js

```diff
@@ -1,11 +1,15 @@
 import { config } from 'dotenv';
+import Redis from 'ioredis';
 import userServices from '../services/user';
+import generateRefresh from '../services/token';
 import successRes from '../utils/successResponse';
 import ErrorResponse from '../utils/errorResponse';
 import encryption from '../helpers/encryption';
 
 config();
 
+const redisClient = new Redis();
```

Comment:
This creates a new default connection (localhost:6379)
- you can pass in a connection url, hence a hosted database: `const redisClient = new Redis(<username:password@redis_stack:port>)`
- you can also make the connection once when the application is started